examples/corpus_token_model.rs

```diff
@@ -80,87 +74,88 @@ pub fn main() {
     println!("-- will lexematize math.")
   }
 
-  let corpus_counts = corpus.catalog_with_parallel_walk(|document| {
-    let (
-      mut paragraph_count,
-      mut word_count,
-      mut overflow_count,
-      mut formula_count,
-      mut citation_count,
-      mut num_count,
-    ) = (0, 0, 0, 0, 0, 0);
-    let mut thread_buffer = String::new();
+  let corpus_counts =
+    corpus.catalog_with_parallel_walk(|document| {
```

Comment:
this one is definitely a strange relint :question: It shouldn't relint a 68-width row, hm